api/airtable.ts

```diff
@@ -43,4 +43,17 @@ export class AirtableClient {
 
     return objects as T[]
   }
+
+  createServiceList(
+    tableName: string,
+    id: string,
+    name: string,
+    description: string,
+    url: string, 
+    addresses?: string[],
+    phoneNumbers?: string[], 
+    email?: string, 
+  ): string {
+      return tableName
+  }
```

Comment:
I think we could make a little more generic? Like just taking in a `tableName` and a list of objects. [Here's Airtable's auto-generating docs for creating a service list for reference.](https://airtable.com/appEQiJGfBTod8EKq/api/docs#javascript/table:services%20lists:create)

Reply:
i tried to make it more generic, lmk

Reply:
Looks good - we could probably switch the return to `records.map(record => record.fields) as T[]` like in my PR though so we don't need to pass in `keys` anymore